source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -91,7 +91,64 @@ After you’ve made an authorisation request for authentication and identity, yo
 
 You can only request user attributes to be returned in the `/userinfo` response. You cannot configure the claims returned in the [ID token][integrate.understand-id-token].
 
-### Replace your example message’s placeholder values
+### Secure your authorisation request parameters with JWT 
+
+You can use JWT-secured OAuth 2.0 (JAR) to encode your request parameters and protect against unwanted tampering with your request. 
+
+GOV.UK One Login follows the [OIDC principles on passing the request objects](https://openid.net/specs/openid-connect-core-1_0.html#RequestObject).
+
+1. Build a request object and sign it using the [private key you created][integrate.generate-key-pair] when setting up your integration with GOV.UK One Login.
+1. Encode the signed request object.
+1. Make a `GET` request replacing `YOU_REQUEST_OBJECT` with your signed and encoded request object.
+
+Use this example to make a `GET` request, replacing your example’s placeholder values: 
```

Comment:
```suggestion
Use this example to make your own `GET` request, replacing the placeholder values: 
```
This might be a little easier to understand as I paused over the 2 references to an example - what do you think?